epilogue-processor/src/main/java/edu/wpi/first/epilogue/processor/SendableHandler.java

```diff
@@ -4,24 +4,29 @@
 
 package edu.wpi.first.epilogue.processor;
 
+import java.util.Optional;
 import javax.annotation.processing.ProcessingEnvironment;
 import javax.lang.model.element.Element;
 import javax.lang.model.type.TypeMirror;
 
 public class SendableHandler extends ElementHandler {
-  private final TypeMirror m_sendableType;
-  private final TypeMirror m_commandType;
-  private final TypeMirror m_subsystemType;
+  private final Optional<TypeMirror> m_sendableType;
+  private final Optional<TypeMirror> m_commandType;
+  private final Optional<TypeMirror> m_subsystemType;
 
   protected SendableHandler(ProcessingEnvironment processingEnv) {
     super(processingEnv);
 
     m_sendableType =
-        lookupTypeElement(processingEnv, "edu.wpi.first.util.sendable.Sendable").asType();
+        Optional.ofNullable(
```

Comment:
Based on the log I was getting, its the asType() call here that is failing. So this probably won't help. We need to make it so asType() isn't called if lookupTypeElement is null.

Reply:
Good catch; I've spent too long in ruby land